std_msgs/Header header
ClosestPointToLineSegment[] closest_point_to_line_segments
